libsnark/gadgetlib1/gadgets/hashes/anemoi/anemoi_parameters.tcc

```diff
@@ -11,744 +11,1101 @@
 
 namespace libsnark
 {
-using ppT = libff::bls12_381_pp;
-using FieldT = libff::Fr<ppT>;
-using BignumT = libff::bigint<FieldT::num_limbs>;
-
-const BignumT anemoi_parameters<ppT>::alpha_inv =
-    BignumT("209743500700504761917790962032743863350762210002110551290414634799"
+const libff::bigint<libff::Fr<libff::bls12_381_pp>::num_limbs>
```

Comment:
You might be able to use `anemoi_parameters<libff::bls12_381_pp>::BignumT` instead of `libff::bigint<libff::Fr<libff::bls12_381_pp>::num_limbs>` here, although the difference is only marginal.

Happy to keep it like this - as you prefer.

Reply:
I like your suggestion. IMO it improves readability. Applied.